currency_rate_update_boi/tests/test_currency_rate_update_boi.py

```diff
@@ -48,18 +48,22 @@ def setUpClass(cls):
         )
         cls.CurrencyRate.search([]).unlink()
 
+    @api.model
+    def _get_no_weekend_date(self, compute_date):
+        if compute_date.weekday() in [5, 6]:
+            days_to_friday = 4 - compute_date.weekday()
+            compute_date -= relativedelta(days=1)
+            return compute_date + timedelta(days=days_to_friday)
```

Comment:
Quindi ritorniamo sempre giovedì?

Reply:
ritorniamo venerdì se il giorno in input è sabato o domerica

Reply:
Non mi torna:
```
>>> compute_date
datetime.datetime(2024, 1, 14, 10, 13, 20, 786868)
>>> days_to_friday = 4 - compute_date.weekday()
>>> compute_date -= relativedelta(days=1)
>>> compute_date
datetime.datetime(2024, 1, 13, 10, 13, 20, 786868)
>>> compute_date + timedelta(days=days_to_friday)
datetime.datetime(2024, 1, 11, 10, 13, 20, 786868)
```
in questo esempio `compute_date` è domenica e viene ritornato giovedì

Reply:
scusami ho scordato di levare `compute_date -= relativedelta(days=1)`